src/core/account/account-reducer.js

```diff
@@ -249,6 +251,18 @@ const accountInner: FatReducer<
     return action.type === 'LOGIN' ? action.payload.username : state
   },
 
+  customTokens(state = {}, action: RootAction): EdgePluginMap<SwapSettings> {
```

Comment:
`SwapSettings` should be `EdgeTokenMap`, which means Flow is pretty broken here.

Reply:
Found a way to fix it, updated the PR.